src/components/PlantingSites/createSite/InstructionsModal.tsx

```diff
@@ -0,0 +1,63 @@
+import strings from 'src/strings';
+import { Box, Theme, Typography } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import { Button, DialogBox } from '@terraware/web-components';
+import { useDocLinks } from 'src/docLinks';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  buttonSpacing: {
+    marginRight: theme.spacing(2),
+  },
+  icon: {
+    width: '150px',
+    height: '150px',
+  },
+}));
+
+export type InstructionsModalProps = {
+  open: boolean;
+  onClose: (dontShowAgain: boolean) => void;
```

Comment:
Should `dontShowAgain` be optional?

Reply:
Good point, yes will make this optional.